package.nls.json

```diff
@@ -84,6 +84,11 @@
     "cmake-tools.configuration.cmake.debugConfig.symbolSearchPath.description": "Visual Studio debugger symbol search paths.",
     "cmake-tools.configuration.cmake.debugConfig.additionalSOLibSearchPath.description": "Paths for GDB or LLDB to search for .so files.",
     "cmake-tools.configuration.cmake.debugConfig.externalConsole.description": "Launch an external console for the program.",
+    "cmake-tools.configuration.cmake.debugConfig.console.description": "Where to launch the debug target. Defaults to 'internalConsole' if not defined.",
+    "cmake-tools.configuration.cmake.debugConfig.console.internalConsole.description": "Output to the VS Code Debug Console. This doesn't support reading console input (ex:'std::cin' or 'scanf').",
```

Comment:
does ex mean example? isn't it e.g. instead of ex ?

Reply:
This is a direct copy from cpptools.  `ex` is a known abbreviation for `example`.